src/EmbeddableObject.js

```diff
@@ -240,8 +240,8 @@ function getIframeSrc(options): string {
         src = src + '&locale=' + locale;
     }
 
-    if (printEnabled) {
-        src = src + "&printEnabled=" + String(printEnabled);
+    if (!printEnabled) {
```

Comment:
this would work for people who dont want print enabled. we are working on a permanent fix, where only people who want print can see it, and people who dont want, will not see it. Thank you for taking up interest in our package, to suggest a fix!